dockerfiles/docker-compose-assets.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+  assets:
+    image: node:8.16
```

Comment:
This is a really old version, we were generating the assets with 10.17 locally

Reply:
8.16 is the one that I've been using all this time. I know people are using different versions, but I don't know which one is using each of us.

I'm happy to update to any other one, but I haven't test them and I don't want to jump into that if it does not add anything to us. It use to break just randomly :smile:

Reply:
You can see our docs mention 10.17 should be used https://github.com/readthedocs/readthedocs.org/pull/7992/files#diff-2f342d78afce7705517e17179ba910d04ba83b37d0d3d4bd0bbacfdec256f6c0L58